scripts/publish-github-release.sh

```diff
@@ -5,7 +5,7 @@ GITHUB_ACCESS_TOKEN=${1:-$GITHUB_ACCESS_TOKEN}
 GITHUB_API_URL_TEMPLATE="https://%s.github.com/repos/%s/%s?access_token=%s%s"
 GITHUB_API_HOST="api"
 GITHUB_UPLOAD_HOST="uploads"
-BINARY_FILE_FILTER="*release.aar"
+BINARY_FILE_FILTER="*.aar"
```

Comment:
We should look for possible impact on xamarin unity build.sh -t=externals before merging.

Reply:
Yeah we have multiple references to `-release.aar` on the .NET repo, we should have the PR ready (don't merge) before merging this.

We might want to check Unity repo too.

Reply:
It's a bug: https://github.com/gradle/gradle/issues/8328
Let's revert gradle to 5.0 for now. Updated masks in the scripts are correct for both versions.